backend/middleware/errorMiddleware.js

```diff
@@ -0,0 +1,16 @@
+const notFound = (req, res, next) => {
+  const error = new Error(`Not Found - ${req.originalUrl}`);
+  res.status(404);
+  next(error);
+};
+
+const errorHandler = (err, req, res, next) => {
+  const statusCode = res.statusCode === 200 ? 500 : res.statusCode;
```

Comment:
Although this ternary function does work (for now), it's functionality is limited as we're only constricted to just status codes 200 and 500. Should we need to support other status codes such in 2XX or 5XX, this portion should be changed.

Reply:
I am not exactly sure how I should correct this. The purpose of line 8 is just to set the res.statusCode to the variable statusCode. Although sometimes, you get a status code of 200 even when there is an error, so in order to distinguish that from an error, I set it to 500.